tests/console/check_system_info.pm

```diff
@@ -81,6 +81,18 @@ sub check_buildid {
     }
 }
 
+# SLE-23605: Python3 module
+sub check_python3_module {
+    my $OS_VERSION = script_output("grep VERSION_ID /etc/os-release | cut -c13- | head -c -2");
+    assert_script_run("SUSEConnect -p sle-module-python3/$OS_VERSION/$ARCH");
+    zypper_call("se python310");
+    zypper_call("in python310");
+    assert_script_run("python3.10 --version | grep Python 3.10.");
+    assert_script_run("zypper lifecycle python310");
```

Comment:
Is there any info need check , such as expire time etc need check it?
Or we still need check old python lifecycle?

Reply:
No needed